packages/familie-dokumentliste/CHANGELOG.md

```diff
@@ -0,0 +1,168 @@
+# Change Log
```

Comment:
Er det litt rar changelog for en helt ny pakke?

Reply:
For mye copy-paste og dårlig forståelse av changeloggen.